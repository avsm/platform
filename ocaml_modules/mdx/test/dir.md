It's possible to run cram tests in separate directories:


```sh dir=../lib
$ [ -e ./mdx.ml ] && exit 1
[1]
```
