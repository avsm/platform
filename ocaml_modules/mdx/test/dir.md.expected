It's possible to run cram tests in separate directories:


```sh dir=../lib
$ ls ./mdx.ml
ls: ./mdx.ml: No such file or directory
[1]
```
